subworkflows/local/contig_coverage.nf

```diff
@@ -0,0 +1,42 @@
+params.rev_reads = null
+params.fwd_reads = null
```

Comment:
This was adapting you "dev" branch code, before even moving it all to nf-core v2. 

It's not implemented yet, like it wasn't in dev.   It should be a sample map when it is, instead of params.

I've added a `TODO: Not yet implemented` to the top of the file